node-module-system/multiply.js

```diff
@@ -0,0 +1,8 @@
+/* in multiply.js define a function with two
+parameters that returns their product. */
+
+function multiply(number1, number2) {
+  return number1 * number2;
+}
+
+export { multiply };
```

Comment:
I prefer the syntax:
`export function multiply(number1, number2) {...}`
instead of the separate
`export { multiply }`